apps/design/frontend/src/features_context.tsx

```diff
@@ -1,79 +1,181 @@
-  PRECINCT_SPLIT_ELECTION_TITLE_OVERRIDE: false,
-  PRECINCT_SPLIT_CLERK_SIGNATURE_IMAGE: false,
-  PRECINCT_SPLIT_CLERK_SIGNATURE_CAPTION: false,
-};
+const userFeatureConfigs = {
+  vx: {
+    ACCESS_ALL_ORGS: true,
+    TABULATION_SCREEN: true,
+    EXPORT_SCREEN: true,
+    ONLY_LETTER_AND_LEGAL_PAPER_SIZES: false,
+    CREATE_ELECTION: true,
+    DELETE_ELECTION: true,
+  },
+  nh: {
+    ACCESS_ALL_ORGS: false,
+    TABULATION_SCREEN: false,
+    EXPORT_SCREEN: false,
+    ONLY_LETTER_AND_LEGAL_PAPER_SIZES: true,
+    CREATE_ELECTION: false,
+    DELETE_ELECTION: false,
+  },
+} satisfies Record<string, UserFeaturesConfig>;
 
-export const NH_ENABLED_FEATURES: FeaturesEnabledRecord = {
-  BALLOT_BUBBLE_SIDE: true,
-  PRECINCT_SPLIT_ELECTION_TITLE_OVERRIDE: true,
-  PRECINCT_SPLIT_CLERK_SIGNATURE_IMAGE: true,
-  PRECINCT_SPLIT_CLERK_SIGNATURE_CAPTION: true,
-};
+const electionFeatureConfigs = {
+  // VX sandbox elections should have not have any state-specific features
+  // enabled
+  vx: {
+    PRECINCT_SPLIT_ELECTION_TITLE_OVERRIDE: false,
+    PRECINCT_SPLIT_CLERK_SIGNATURE_IMAGE: false,
+    PRECINCT_SPLIT_CLERK_SIGNATURE_CAPTION: false,
+  },
 
-const FeaturesContext = createContext<FeaturesEnabledRecord>(
-  DEFAULT_ENABLED_FEATURES
-);
+  nh: {
+    PRECINCT_SPLIT_ELECTION_TITLE_OVERRIDE: true,
+    PRECINCT_SPLIT_CLERK_SIGNATURE_IMAGE: true,
+    PRECINCT_SPLIT_CLERK_SIGNATURE_CAPTION: true,
+  },
+} satisfies Record<string, ElectionFeaturesConfig>;
 
-export function useFeaturesContext(): FeaturesEnabledRecord {
+const FeaturesContext = createContext<FeaturesConfig | undefined>(undefined);
+
+function useFeatures(): FeaturesConfig {
   return assertDefined(
     useContext(FeaturesContext),
     'useFeatures must be used within a FeaturesProvider'
   );
 }
 
+export function useUserFeatures(): UserFeaturesConfig {
+  const features = useFeatures();
+  return features.user;
+}
+
+export function useElectionFeatures(): ElectionFeaturesConfig {
+  const features = useFeatures();
+  return assertDefined(
+    features.election,
+    'Must pass electionId to FeaturesProvider to access election features'
+  );
+}
+
+export function UserFeaturesProvider({
+  children,
+}: {
+  children: React.ReactNode;
+}): JSX.Element | null {
+  const getUserQuery = getUser.useQuery();
+  if (!getUserQuery.isSuccess) {
+    return null;
+  }
+
+  // TODO: Eventually we'll want to map user org IDs to feature configs (likely
+  // on the backend), but for now we just use a flag to differentiate between
+  // VX and NH users.
+  const userFeatures = getUserQuery.data.isVotingWorksUser
+    ? userFeatureConfigs.vx
+    : userFeatureConfigs.nh;
+
+  return (
+    <FeaturesContext.Provider value={{ user: userFeatures }}>
+      {children}
+    </FeaturesContext.Provider>
+  );
+}
+
 interface FeaturesProviderProps {
   children: React.ReactNode;
-  electionId?: ElectionId;
+  electionId: ElectionId;
 }
 
 export function FeaturesProvider({
```

Comment:
Might be helpful to copy or adapt the comments from `UserFeature` and `ElectionFeature` enums to explain when to use `FeaturesProvider` vs `UserFeaturesProvider`. Something simple like "Use `UserFeaturesProvider` for screens where there isn't a specific election" could help differentiate at a glance in vscode autocomplete.

Reply:
Good idea!